mod/network.php

```diff
@@ -883,17 +884,19 @@ function networkThreadedView(App $a, $update = 0) {
  * @param App $a The global App
  * @return string Html of the networktab
  */
-function network_tabs(App $a) {
+function network_tabs(App $a)
+{
 	// item filter tabs
 	/// @TODO fix this logic, reduce duplication
 	/// $a->page['content'] .= '<div class="tabs-wrapper">';
-
 	list($no_active, $all_active, $postord_active, $conv_active, $new_active, $starred_active, $bookmarked_active, $spam_active) = network_query_get_sel_tab($a);
+
 	// if no tabs are selected, defaults to comments
-	if ($no_active=='active') $all_active='active';
+	if ($no_active == 'active') {
+		$all_active = 'active';
+	}
 
-	$cmd = (($datequery) ? '' : $a->cmd);
-	$len_naked_cmd = strlen(str_replace('/new','',$cmd));
```

Comment:
Is "datequery" and "len_naked_cmd" never used?

Reply:
Nope...

Reply:
"Nope" means "yes"? (Think of double negation) :smile:

Reply:
It does, I think? Those variables aren’t used ever.

Reply:
It does, I think? Those variables aren’t used ever.

Reply:
`$datequery` (and also `$cid`) are defined in [networkThreadedView()](https://github.com/MrPetovan/friendica/blob/58ac3532b74017802e4027400a3fe6b4eb76efe2/mod/network.php#L518-L540)

Could be that the code was splitted?

Reply:
Maybe, but it still doesn't make sense in `network_tabs()`.